op/model.py

```diff
@@ -256,6 +265,7 @@ def __init__(self):
 
     def _run(self, *args):
         result = run(args + ('--format=json',), stdout=PIPE, stderr=PIPE, check=True)
+
```

Comment:
This extra line doesn't seem intentional or worthwhile here. Probably a leftover.